### DISCUSSION TITLE
Is it possible to test core wikitext-macros using the new $:/tags/wiki-test-spec

### DISCUSSION THREAD

**Discussion by pmario** [TiddlyWiki/TiddlyWiki5#8400] [General]
@Jermolene -- Is it possible to test eg: `<<list-links ...>>` macro using the "new" wikitext test-spec?

eg: 
```
<<list-links filter:"[tag[testx]]">>
---
```

produces an output: `Exptected '<hr>' to be '',` ... So is there a possibility to "Import" core wikitext macros?

**Comment by Jermolene** [comment]
Hi @pmario to use wikitext tests to test macros like `<<list-links>>` would require some further work. The testcase widget itself does already support pulling in tiddlers from the host wiki, but we don't have similar functionality for the testcase runner in `$:/plugins/tiddlywiki/jasmine/run-wiki-based-tests.js`.

We would need to extend the compound tiddler format with some new syntax, hopefully being able to take inspiration from the existing `<$data>` widget.

**Reply by pmario** [reply]
Thx for the info
